geolocator_apple/lib/src/types/apple_settings.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:geolocator_platform_interface/geolocator_platform_interface.dart';
+import 'activity_type.dart';
+
+/// Represents different iOS specific settings with which you can set a value
+/// other then the default value of the setting.
+class AppleSettings extends LocationSettings {
+  /// Initializes a new [IosSpecificSettings] instance with default values.
```

Comment:
Use correct type in comments:

```suggestion
  /// Initializes a new [AppleSettings] instance with default values.
```